packages/prices/src/prices.ts

```diff
@@ -0,0 +1,31 @@
+import { PriceSpecification } from "@js-ligo/vocab";
+
+export interface DynamicPrice {
```

Comment:
Is there a way we can represent this `DynamicPrice` inside the `PriceSpecification`? See http://wiki.goodrelations-vocabulary.org/Cookbook/Pricing for some examples of this pricing model. An `Offer` has a list of `PriceSpecification` attached to it. So each date could be it's own `PriceSpecification` with the `validFrom` and `validThrough` being the same day.

Reply:
Thanks, I'll look more into it.